features/expectations/body_text_example.feature

```diff
@@ -46,6 +46,5 @@ Feature: Body - text example
       Orange, strawberry, banana?
       Dog, cat, mouse!
       """
-    Then field "body" is valid
-    And Request or Response is valid
-
+    Then field "body" MUST valid
```

Comment:
```suggestion
    Then field "body" MUST be valid
```

Reply:
Assertion vocabulary replaced to use more readable `is valid` and `is NOT valid` terms (as previously).